src/interfaces/coinbase/IAggregatorV3SourceCoinbase.sol

```diff
@@ -7,10 +7,10 @@ interface IAggregatorV3SourceCoinbase {
     function latestRoundData(string memory ticker)
         external
         view
-        returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound);
```

Comment:
Additionally, we can simplify the return values, given that they don’t match the Chainlink Aggregator interface anymore with the ticker argument.